fairlearn/metrics/_base_metrics.py

```diff
@@ -8,11 +8,13 @@
 
 import numpy as np
 import sklearn.metrics as skm
-from typing import Any
+from typing import Any, Optional, TypeVar
 
 from ._metric_frame import check_consistent_length
 from fairlearn.utils._input_manipulations import _convert_to_ndarray_and_squeeze
 
+ArrayLike = TypeVar("ArrayLike")
```

Comment:
Thoughts on this? I think if adopted across the codebase, we could tamp down on the number of warnings in the doc build